.gitignore

```diff
@@ -42,3 +42,5 @@ inventory_url.yaml
 # static/version.txt
 
 .idea/
+
+analytics/
```

Comment:
Looks like this got committed with your `gofmt` fix... I'm assuming it wasn't intentional :)

Reply:
Yep. I'll get that out for next commit.